.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
+\#*\#
```

Comment:
Not a fan of editor specific stuff.

Reply:
Isn't this whole project editor-specific? I can remove it, but isn't it an Emacs-specific extension?

Reply:
It is, but that does not change the fact anyone could modify Emacs to use different file suffixes for example.

Reply:
Makes sense. Done.